src/screens/OParl/OParlOrganisationsScreen.tsx

```diff
@@ -75,9 +75,8 @@ export const OParlOrganizationsScreen = ({ navigation }: Props) => {
     <SafeAreaViewFlex>
       <FlatList
         data={listData}
-        renderItem={({ item }) => (
-          <OParlPreviewComponent data={item} key={item.id} navigation={navigation} />
-        )}
+        keyExtractor={(item, index) => `index${index}-id${item.id}`}
```

Comment:
This is how we use `keyExtractor` in many parts of the code. How about creating a helper function?

For example:
```
const keyExtractor = (item, index) => `item${item?.id || item}-index${index}`;
```
```
<FlatList
  keyExtractor={keyExtractor}
  ...
/>
```

Reply:
That's not what I meant by the latest update. Currently the repetition problem in the code is not solved. I mean creating a single `keyExtrator` helper function and using that function everywhere.

For Example: /src/helper/keyExtrator.ts
```
export const keyExtractor = (item, index) => `item${item?.id || item}-index${index}`;
```

I suggested to use all keyExtrators used in the code through this function.

Reply:
that will not work for every list, as the items can be different from component to component.